### PR TITLE
(project) Adds Scattering/buckshot shotgun shells that spreads

### PR DESCRIPTION
**Pull request by SplinterGP**
This adds a shotgun shell that spreads, since i noticed none of the shotgun ammo spreads (the shell one is a pelletshot, when you fire its actually a single pellet, the difference of it to a slug is that it counts as 6 when it hits and loses damage per travel), so this is basically a type of ammo that actually spreads, it has no set damage, no balancing stuff yet done, only it's work is made with an already existing scattering shots, these also use the pellets, but i was considering into turning it into 6 shots rather than 6 shots that each causes 6 or 4 pellets (for easier proccessing and lesser logs if needed) 

How Pellets work mechanically: 
pellets working

so pellets are basically just a single shot that is better on close range

Mechanically that's how the buckshots/scatter ones will work:
Scattershot working

So far it still uses the pellets too so its 6 shots that are 6 pellets, they aren't balanced yet so its just a test

Ingame test pics:
scattershot test1

scattershot test2

It also has the yellow bullet on the test for seeing it better, since it is a test and i want to see whats happening since the actual bullet sprites are very stealthy

### CODE REVIEW

**Comment by SplinterGP** [comment]
good to merge now, balanced and all

File: code/modules/projectiles/guns/projectile/shotgun_yw.dm [hunk 1]
```diff
@@ -50,4 +50,36 @@
...
+		/obj/item/projectile/bullet/shotgun/scatterprojectile = 6
+		)
+/obj/item/projectile/bullet/shotgun/scatterprojectile
+	name = "pellet"
+	fire_sound = 'sound/weapons/Gunshot_shotgun.ogg'
+	damage = 13
+
+/obj/item/weapon/storage/box/scattershot
+	name = "box of shotgun scatter shells"
+	desc = "It has a picture of a gun and several warning symbols on the front.<br>WARNING: Live ammunition. Misuse may result in serious injury or death."
```
Comment (Repede) [inline code/modules/projectiles/guns/projectile/shotgun_yw.dm]: forgot to close the \<br\> tag here

Comment (SplinterGP) [reply code/modules/projectiles/guns/projectile/shotgun_yw.dm]: Aint a tag, it just jumps a line, like ingame writing [br]

Comment (SplinterGP) [reply code/modules/projectiles/guns/projectile/shotgun_yw.dm]: I believe so, copied from the other ammo box desc so i think thats it

Comment (KillianKirilenko) [reply code/modules/projectiles/guns/projectile/shotgun_yw.dm]: Yeah br doesn't need an end tag, unlike most other html. It's just a linebreak.

Comment (Repede) [reply code/modules/projectiles/guns/projectile/shotgun_yw.dm]: I am dumb yeah

**Pull request merged**